test/02-investigate.t

```diff
@@ -87,6 +87,9 @@ client-get-job() {
         30032)
             echo '{"job": { "test": "vim:investigate:retry", "result": "passed", "settings": {"OPENQA_INVESTIGATE_ORIGIN": "3003"} } }'
             ;;
+	10035)
```

Comment:
Please don't use tabs.

I think we have to change our checkstyle to also check `test/*.t`. However, shfmt doesn't seem to check for tabs...

Reply:
Actually, shfmt does reformat tabs. but it doesn't see `test/*.t` as shell files

Reply:
Fixed by #382